src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -273,29 +254,22 @@ def get_string_dtg(obs_time_utc):
 
 
 def init_obs_loc():
-    # V2     ('brightnessTemperature', "ObsValue"): [],
-    # V2     ('brightnessTemperature', "ObsError"): [],
-    # V2     ('brightnessTemperature', "PreQC"): [],
-    # V2     ('fieldOfViewNumber', 'MetaData'): [],
-    # V2     ('solarZenithAngle', 'MetaData'): [],
-    # V2     ('solarAzimuthAngle', 'MetaData'): [],
-    # V2     ('sensorZenithAngle', 'MetaData'): [],
-    # V2     ('sensorAzimuthAngle', 'MetaData'): [],
     obs = {
-        ('brightness_temperature', "ObsValue"): [],
-        ('brightness_temperature', "ObsError"): [],
-        ('brightness_temperature', "PreQC"): [],
-        ('satelliteId', 'MetaData'): [],
-        ('channelNumber', 'MetaData'): [],
+        ('brightnessTemperature', "ObsValue"): [],
+        ('brightnessTemperature', "ObsError"): [],
+        ('brightnessTemperature', "PreQC"): [],
+        ('satelliteIdentifier', 'MetaData'): [],
+        ('sensorChannelNumber', 'MetaData'): [],
         ('latitude', 'MetaData'): [],
         ('longitude', 'MetaData'): [],
-        ('datetime', 'MetaData'): [],
-        ('scan_position', 'MetaData'): [],
-        ('solar_zenith_angle', 'MetaData'): [],
-        ('solar_azimuth_angle', 'MetaData'): [],
-        ('sensor_zenith_angle', 'MetaData'): [],
-        ('sensor_view_angle', 'MetaData'): [],
-        ('sensor_azimuth_angle', 'MetaData'): [],
+        ('dateTime', 'MetaData'): [],
```

Comment:
There are a very large number of lines with hard-coded group names of ```MetaData```, ```ObsValue```, ```ObsError```, and ```PreQc```.  If you review my python code, we can take simple care of making a set of 4 variables to query what group names are in IODA standard to make future hard-coded names less necessary.  Please review the 4 lines of code starting at [amdar_bufr2ioda.pyL84](https://github.com/JCSDA-internal/ioda-converters/blob/7b46bc7f4401e539639a6e1f8142ef706cc1d8b7/src/conventional/amdar_bufr2ioda.py#L84).  If you capture the group names and re-use them later, then literally no code changes are needed if some future IODA version changes group names.

Reply:
yes would like to defer to another issue and pull request

Reply:
done